### ISSUE TITLE
Header dependency issue on Fedora Workstation 37

### ISSUE THREAD

**Issue by PhoenixCausesOof** [Frogging-Family/wine-tkg-git#962]
### The problem

When trying to create wine builds (not necessarily using wine-tkg), I'm having an issue where, despite obviously having a dependency installed, its headers are not detected. For example, I get this error message while configuring: 

`checking for gst/gst.h... no`
`configure: error: gstreamer-1.0 base plugins 32-bit development files not found, GStreamer won't be supported.` 

DNF says otherwise:
`$ dnf list installed | grep "gstreamer"`
![image](https://user-images.githubusercontent.com/63979557/221247870-78bd5378-1d20-4b9f-81cb-149c2527b670.png)
 
Please do note that the package `gstreamer1-devel` provides the header mentioned above (I checked it with `dnf provides`).

This also happens similarly while building with wine_wayland_driver patch, but with the header `drm.h` instead.

![image](https://user-images.githubusercontent.com/63979557/221251835-338aa6b5-088b-497d-bc0e-34b639209aba.png)

And once again, I have the necessary package installed (`libdrm-devel` provides `drm.h`).

`$ dnf list installed | grep "libdrm"`
![image](https://user-images.githubusercontent.com/63979557/221252202-14d7f7d6-25fb-4d99-8ba1-9a4b7de4fd2e.png)

### Diagnostic

So, the issue is that the headers' location are incorrectly `include`d. I have `/usr/include/gstreamer-1.0/gst/gst.h` but not `/usr/include/gst/gst.h`. As well as `/usr/include/libdrm/drm.h` but not `/usr/include/drm.h`. The header files are there, but they are simply one level below the include directory.

### Solution

Apparently, I need it to look for the headers recursively, or use `pkg-config` (according to [this](https://stackoverflow.com/questions/510098/g-include-all-usr-include-recursively) post). How do I do that with wine-tkg? 

P.S. sorry for making this post look like a mess.

**Comment by PhoenixCausesOof** [comment]
Update: I just had to set the environment variables DRM_LIBS, DRM_CFLAGS, GBM_LIBS, GBM_CFLAGS and GSTREAMER_LIBS, GSTREAMER_CFLAGS to their `pkg-config` respectives. Also set PKG_CONFIG_PATH to `/usr/lib32/pkgconfig` (apparently there was something with that too). 

Oh, and some 32-bit packages are not resolved by the build script (wayland-devel.i686, libxkbcommon-devel.i686, mesa-libgbm-devel.i686 and libdrm.i686), though they are only necessary for the wayland driver, and not the plain one.

**Comment by kedras** [comment]
Hi can you pls describe more in detail what you set to make it work? Running into same issue and trying to wrap my head around it.

Thank you

**Comment by Melechtna** [comment]
> Update: I just had to set the environment variables DRM_LIBS, DRM_CFLAGS, GBM_LIBS, GBM_CFLAGS and GSTREAMER_LIBS, GSTREAMER_CFLAGS to their `pkg-config` respectives. Also set PKG_CONFIG_PATH to `/usr/lib32/pkgconfig` (apparently there was something with that too).
> 
> Oh, and some 32-bit packages are not resolved by the build script (wayland-devel.i686, libxkbcommon-devel.i686, mesa-libgbm-devel.i686 and libdrm-devel.i686), though they are only necessary for the wayland driver, and not the plain one.

TLDR: I figured it out

Bro, at least explain how you got the answer, because me and this other guy are sitting in the same boat.
